src/__tests__/react-plotly.test.js

```diff
@@ -135,32 +135,6 @@ describe('<Plotly/>', () => {
           .catch(err => done.fail(err));
       });
 
-      test('clear event handlers on newPlot', done => {
```

Comment:
this test was way too implementation-specific and now depends on the Plotly.js version.